src/screens/MeetScreen.tsx

```diff
@@ -1,14 +1,49 @@
-import { View, Text } from 'react-native';
+import { StyleSheet, ImageSourcePropType } from 'react-native';
 import { SafeAreaView } from 'react-native-safe-area-context';
+import MeetSearchInput from '../components/MeetSearchInput';
+import MeetList from '../components/MeetList';
+import InitMeetBtn from '../components/InitMeetBtn';
+import MeetLogo from '../components/MeetLogo';
+import MeetBtn from '../components/MeetBtn';
+import { useState, useEffect } from 'react';
+import { TempMeetList } from '../data/TempMeetList';
+
+export interface MeetListArr {
+  id: number;
+  meetImg: ImageSourcePropType;
+  meetName: string;
+  meetIntroduce: string;
+  isJoin: boolean;
+}
 
 const MeetScreen = () => {
+  const [hasMeet, setHasMeet] = useState(false);
+  const [resultMeetList, setResultMeetList] = useState<MeetListArr[]>([]);
+
+  useEffect(() => {
+    const temp = TempMeetList.filter(item => item.isJoin);
+    temp && setHasMeet(true);
```

Comment:
useEffect 안에서는 이런 식의 코드보다 if 문이 나을듯 해요
```suggestion
    if (temp) {
      setHasMeet(true);
    }
```

Reply:
추가로 hasMeet 은 state 보다는 함수로 빼는게 나아요. 이런 식으로 state 로 만든다면 혹시 개발자가 실수하면 hasMeet 이랑 temp 의 존재 여부가 다를 수도 있으니까요

Reply:
오호,, `jsx내부에서는 statement를 수정하는 행위가 있으면 안 된다` 멤모,,

Reply:
` temp && setHasMeet(true);` 얘는 expression. 지금 상황에서 쓰일 건 if문과 같은 statement다.